syntax = "proto3";
package buildorders;

service BuildOrderService {
  rpc SendOrders (stream BuildOrder) returns (StatusResponse);
}

message BuildOrder {
  int32 id_order = 1;
  string address = 2;
  string work_stages = 3;
  string work_prices = 4;
  string materials = 5;
  string material_quantities = 6;
  string material_prices = 7;
}

message StatusResponse {
  string status = 1;
}
